src/news-display/gridView.js

```diff
@@ -0,0 +1,58 @@
+import { gridPage, newsPressData } from "../app.js";
+import { checkShowPageButton } from "./handlePageButton.js";
+import { initNewsPressData } from "./initNewsPressData.js";
+
+const PAGE_COUNT = 24;
+
+// let gridPage = 1;
+let shuffledNewsPressData = [];
+
+const shuffleArray = (array) => array.sort(() => Math.random() - 0.5);
+
+const fillEmptyCells = (currentNewsPressData, newsPressGrid) => {
+    if (currentNewsPressData.length < 24) {
+        const count = currentNewsPressData.length;
+        for (let i = 0; i < 24 - count; i++) {
+            const $li = document.createElement("li");
+            $li.className = "news-press-item";
+            newsPressGrid.appendChild($li);
+        }
+    }
```

Comment:
```suggestion
while(currentNewsPressData.length<24) {
    const $li = document.createElement("li");
    $li.className = "news-press-item";
    newsPressGrid.appendChild($li);
}
```

if 문과 for문을 while 문으로 축약할 수 있을 것 같아요!